app/models/mission/MissionTable.scala

```diff
@@ -215,6 +225,40 @@ object MissionTable {
     regionalMissions.sortBy(rm => (rm.regionId, rm.missionId))
   }
 
+  /**
+    * Return a list of missions for a specific user
+    *
+    * @param userId User id
+    * @return
+    */
+  def selectMissions(userId: UUID): List[AuditMission] = db.withSession { implicit session =>
+    // gets all the missions that correspond to the user
+    val userMissions = for {
```

Comment:
@paarig check out `StreetEdgeTable.scala` line 80 for an example of a cleaner way to write multiple inner joins.

Reply:
Fixed